### PR TITLE
Code repetition has been removed in package "dep-tree-js". Made more readable.

### PR DESCRIPTION
**Pull request by aykutkardas**
Code repetition has been removed in package "dep-tree-js". Made more readable.

### CODE REVIEW

File: packages/dep-tree-js/index.js [hunk 1]
```diff
@@ -3,22 +3,19 @@ const path = require("path");
 const fs = require("fs");
 const builtInPackages = require("module").builtinModules || [];
 
+const avaibleExtensions = [".js", ".jsx", ".md", ".mdx", ".ts", ".tsx", ".vue"];
```
Comment (asadm) [inline packages/dep-tree-js/index.js]: @aykutkardas can you please change this variable's name to `availableExtensions`.

Comment (aykutkardas) [reply packages/dep-tree-js/index.js]: Oh sorry.

Comment (aykutkardas) [reply packages/dep-tree-js/index.js]: I sent it along with a change. @asadm

**Comment by asadm** [comment]
Awesome thanks!

**Pull request merged**